package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "@openzim/libzim",
+  "main": "dist/index.js",
+  "types": "dist/index.d.js",
   "version": "2.4.4",
   "description": "Libzim bindings for NodeJS",
   "scripts": {
     "clean": "rm -rf dist build/native/build",
     "tsc": "tsc",
```

Comment:
We could probably remove this, no longer required, just use `src/index.js` directly.

Reply:
No opinion on this...